distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/opensearch/OpenSearchTestCommons.java

```diff
@@ -29,27 +37,66 @@
 
     public static final String OPENSEARCH_FACTORY_PID = "OpenSearchSource";
 
-    public static Map<String, Object> getOpenSearchSourceProperties(String sourceId, String openSearchUrl, ServiceManager serviceManager) {
+    public static final AbstractIntegrationTest.DynamicPort HTTPS_PORT =
+            new AbstractIntegrationTest.DynamicPort("org.codice.ddf.system.httpsPort", 2);
+
+    public static final AbstractIntegrationTest.DynamicUrl SERVICE_ROOT =
+            new AbstractIntegrationTest.DynamicUrl(AbstractIntegrationTest.DynamicUrl.SECURE_ROOT,
+                    HTTPS_PORT,
+                    "/services");
+
+    public static final AbstractIntegrationTest.DynamicUrl REST_PATH =
+            new AbstractIntegrationTest.DynamicUrl(SERVICE_ROOT, "/catalog/");
+
+    public static final AbstractIntegrationTest.DynamicUrl OPENSEARCH_PATH =
+            new AbstractIntegrationTest.DynamicUrl(REST_PATH, "query");
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractIntegrationTest.class);
+
+    public static Map<String, Object> getOpenSearchSourceProperties(String sourceId,
+            String openSearchUrl, ServiceManager serviceManager) {
         Map<String, Object> openSearchSourcePropertes = new HashMap<>();
-        openSearchSourcePropertes.putAll(serviceManager.getMetatypeDefaults(OPENSEARCH_SYMBOLIC_NAME, OPENSEARCH_FACTORY_PID));
+        openSearchSourcePropertes.putAll(serviceManager.getMetatypeDefaults(OPENSEARCH_SYMBOLIC_NAME,
+                OPENSEARCH_FACTORY_PID));
         openSearchSourcePropertes.put("shortname", sourceId);
         openSearchSourcePropertes.put("endpointUrl", openSearchUrl);
         return openSearchSourcePropertes;
     }
 
-    public static ValidatableResponse executeOpenSearch(String format, String... query) {
+    /**
+     * Gets the validatble response of the open search query
+     * When admin is set to true, there is additional authentication specifications
+     * for setting up as an admin
+     *
+     * @return Returns the ValidatableResponse object that will be used for validation
```

Comment:
✏️  `@return` should be last. Running the formatter on the file may fix that and also get rid of the extra spaces between `@param` and their descriptions.